packages/calypso-e2e/src/lib/base-block.ts

```diff
@@ -0,0 +1,20 @@
+import { ElementHandle } from 'playwright';
+
+/**
+ * Base class for the Block type objects.
+ */
+export class BaseBlock {
```

Comment:
Should we remove this file from the commit because it is no longer needed?

Reply:
I thought I had removed it, my bad.

Reply:
Done!